include/msgpack/v1/sbuffer.hpp

```diff
@@ -71,7 +71,9 @@ class sbuffer {
         if(m_alloc - m_size < len) {
             expand_buffer(len);
         }
-        std::memcpy(m_data + m_size, buf, len);
+        if(buf) {
+            std::memcpy(m_data + m_size, buf, len);
+        }
         m_size += len;
```

Comment:
Should this only be inside the `if`?

Reply:
same as https://github.com/msgpack/msgpack-c/pull/890#discussion_r447380795

Reply:
@ygj6 , could you apply the same fix as #890 ?